src/main/groovy/com/ullink/NuGetPush.groovy

```diff
@@ -28,7 +26,6 @@ class NuGetPush extends BaseNuGet {
 
         if (serverUrl) args '-Source', serverUrl
         if (apiKey) args '-ApiKey', apiKey
-        if (timeout) args '-Timeout', timeout
```

Comment:
@ngyukman To me it doesn't seem like it would have the same end result, no? The existing one stops "gracefully" nuget itself, but the new one will just kill nuget.exe, no?

Reply:
yes, but I don't see the actual difference ?
probably nuget kills the http request itself when timeout